client/lib/billing/app/views/overlays/billing.js

```diff
@@ -0,0 +1,12 @@
+import OverlayView from 'tahi/views/overlay';
+
+export default OverlayView.extend({
+  templateName: 'overlays/billing',
+  layoutName:   'layouts/overlay',
+  cardName: 'billing',
+
+  didInsertElement: function() {
+    var choice = $(".payment-method .select2-container").select2("val");
+    this.get("controller").set("selectedPaymentMethod", choice);
```

Comment:
Since the `ident` is clearly defined and task is associated to its questions, you should be able to use `find` with the `question`'s `ident`, which I believe is, `billing.payment_method`. That should return the currently selected payment method.